specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,205 @@
+        "timeUnixNano":"1581452773000000789",
+        "attributes": [
+        {
+            "key": "event.domain",
+            "value": {
+            "stringValue": "browser"
+            }
+        },
+        {
+            "key": "event.name",
+            "value": {
+            "stringValue": "page_view"
+            }
+        },
+
+        {
+            "key": "event.data",
+            "value": {
+                "kvlistValue": [
+                    {
+                        "key": "type",
+                        "value": {
+                        "intValue": "0"
+                        }
+                    },
+                    {
+                        "key": "url",
+                        "value": {
+                        "stringValue": "https://www.guidgenerator.com/online-guid-generator.aspx"
+                        }
+                    },
+                    {
+                        "key": "referrer",
+                        "value": {
+                        "stringValue": "https://wwww.google.com"
+                        }
+                    },
+                    {
+                        "key": "title",
+                        "value": {
+                        "stringValue": "Free Online GUID Generator"
+                        }
+                    },
+                ]
+            }
+        }
+        ]
+    }
+```
+
+</details>
+
+### PageNavigationTiming
+
+**Event name**:`page_navigation_timing`
+
+This event describes the timing metrics of a page navigation as provided by
+`PerformanceNavigationTiming` Performance API.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| fetchStart | long | || Recommended |
+| unloadEventStart | long | || Recommended |
+| unloadEventEnd | long | || Recommended |
+| domInteractive | long | || Recommended |
+| domContentLoadedEventStart | long | || Recommended |
+| domContentLoadedEventEnd | long | || Recommended |
+| domComplete | long | || Recommended |
+| loadEventStart | long | || Recommended |
+| loadEventEnd | long | || Recommended |
+| firstPaint | long | || Recommended |
+| firstContentfulPaint | long | || Recommended |
+
+### ResourceTiming
+
+**Event name**:`resource_timing`
+
+This event describes the timing metrics as provided by `PerformanceResourceTiming`
+Performance API.  These metrics are related to fetching a resource, such as
+XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+|fetchStart | long | || Recommended |
+|domainLookupStart | long | || Recommended |
+|domainLookupEnd | long | || Recommended |
+|connectStart | long | || Recommended |
+|secureConnectionStart | long | || Recommended |
+|connectEnd | long | || Recommended |
+|requestStart | long | || Recommended |
+|responseStart | long | || Recommended |
+|responseEnd | long | || Recommended |
+
+### Exception
+
+**Event name**:`exception`
```

Comment:
Why is this redefined?  Shouldn't this just leverage existing exception specification, e.g. https://github.com/open-telemetry/semantic-conventions/blob/main/specification/logs/semantic_conventions/exceptions.md